addons/mail/static/src/model/misc.js

```diff
@@ -49,11 +49,13 @@ export function isOne(Model, fieldName) {
 export function isMany(Model, fieldName) {
     return Model._.fieldsMany.get(fieldName);
 }
+/** @param {Record} record */
```

Comment:
I don't think this doc is correct, param of function should be "any" otherwise there is no point to check if it's a record.